tests/e2e/messages-content.js

```diff
@@ -0,0 +1,209 @@
+const utils = require('../utils'),
+  helper = require('../helper'),
+  common = require('../page-objects/common/common.po');
+
+describe('Display message', () => {
+  'use strict';
+
+  const ALICE = {
+    _id: 'alice-contact',
+    reported_date: 1,
+    type: 'person',
+    name: 'Alice Alison',
+    phone: '+447765902001',
+  };
+
+  const CONTACTS = [ALICE];
+
+  beforeAll(done => {
+    protractor.promise
+      .all(CONTACTS.map(utils.saveDoc))
+      .then(done)
+      .catch(done.fail);
+  });
+
+  afterEach(done => {
+    utils.resetBrowser();
+    utils
+      .deleteAllDocs()
+      .then(done);
+  });
+
+  afterAll(done => {
+    utils.afterEach(() => {
+      done();
+    });
+  });
+
+  const messageInList = identifier => {
+    return '#message-list li[data-record-id="' + identifier + '"]';
+  };
+
+  const smsMsg = key => {
+    return 'Hello ' + key + ' this is a test SMS';
+  };
+
+  const openSendMessageModal = () => {
+    helper.waitElementToBeClickable(
+      element(by.css('.general-actions .send-message'))
+    );
+    helper.clickElement(element(by.css('.general-actions .send-message')));
+    helper.waitElementToPresent(element(by.id('send-message')), 5000);
+    helper.waitElementToBeVisible(element(by.id('send-message')), 5000);
+  };
+
+  const findSelect2Entry = (selector, expectedValue) => {
+    return element
+      .all(by.css('.select2-results__option' + selector))
+      .filter(item => {
+        return item
+          .getText()
+          .then(text => {
+            return text === expectedValue;
+          })
+          .catch(err => {
+            // item may have been detached from the page, so whatever it's invalid,
+            // we ignore it. Log the error just for kicks.
+            console.log('Caught and ignoring an error trying to getText', err);
+          });
+      });
+  };
+
+  const searchSelect2 = (
```

Comment:
A lot of this test includes helper code seemingly copied from the existing send-message e2e test. 

It would make more sense, since this test is also about the same page, to just rename send-message to be more generally about the messages page and add your test to that.